cfn-resources/resource-policy/test/inputs_2_create.template.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
looks identical to the `inputs_1`, can we have a test case for `_invalid` too? To check the error message?

Reply:
I don't think we can test for invalid inputs in the contract tests here.

Reply:
We do it in the `cluster` resource [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/cluster/inputs/inputs_1_invalid.json)